src/store/transactions/withdraw/withdraw.thunks.ts

```diff
@@ -252,7 +252,7 @@ function withdraw(
 
 function handleTransactionSuccess(dispatch: AppDispatch, accountIndex: string) {
   dispatch(openSnackbar("Transaction submitted"));
-  dispatch(push(`/accounts/${accountIndex}`));
```

Comment:
This is not going to prevent the user to reach the withdraw from copy & paste a url and clicking on submit, also I don' like the `replace` approach as it breaks the normal flow of the browser.

The ideal flow should be to check if the withdraw has already been submitted by checking if it's still in `localStorage` or something like this, as I think that we clear them once they are mined in L1. But I'm not sure, we can meet on monday to talk about this :)